src/measure_theory/group/fundamental_domain.lean

```diff
@@ -202,6 +232,16 @@ by simpa only [set_lintegral_one] using h.set_lintegral_eq_tsum (λ _, 1) t
   μ t = 0 :=
 by simp [measure_eq_tsum h, ht, hts]
 
+@[to_additive measure_eq_card_smul_of_vadd_eq_self]
+lemma measure_eq_card_smul_of_smul_eq_self [finite G]
+  (h : is_fundamental_domain G s μ) (t : set α) (ht : ∀ g : G, g • t = t) :
+  μ t = nat.card G • μ (t ∩ s) :=
```

Comment:
Please add a docstring. BTW, what is the current convention about `nat.card` vs `fintype.card`?

Reply:
I've added a docstring.

I believe we favour `finite` over `fintype` (and given this I must use `nat.card` since I don't have a `fintype` instance).